manifests/cf-manifest/deployments/060-aws-broker.yml

```diff
@@ -62,6 +62,33 @@ jobs:
                     db_subnet_group_name: (( grab terraform_outputs.rds_broker_dbs_subnet_group ))
                     vpc_security_group_ids:
                       - (( grab terraform_outputs.rds_broker_dbs_security_group_id ))
+                - id: "9b882524-ab58-4c18-b501-d2a3f4619104"
+                  name: "M-dedicated-9.5"
+                  description: "20GB Storage, Dedicated Instance, 20 Concurrent Connections. Postgres Version 9.5"
```

Comment:
I think the server should be able to take _many more_ than 20 concurrent connections...